server/journeys/screen/start/controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express'
+import { SanitisedError } from '../../../sanitisedError'
+import CsipApiService from '../../../services/csipApi/csipApiService'
+import PrisonerSearchService from '../../../services/prisonerSearch/prisonerSearchService'
+
+export class StartJourneyController {
+  constructor(
+    private readonly csipService: CsipApiService,
+    private readonly prisonerSearchService: PrisonerSearchService,
+  ) {}
+
+  redirectWithCsipData =
+    (url: string) =>
+    async (req: Request, res: Response): Promise<void> => {
+      const { csipRecordId, journeyId } = req.params
+
+      try {
+        const csip = await this.csipService.getCsipRecord(req, csipRecordId as string)
+        req.journeyData.csipRecord = csip
+        req.journeyData.saferCustodyScreening = {}
+
+        const prisoner = await this.prisonerSearchService.getPrisonerDetails(req, csip.prisonNumber as string)
+        req.journeyData.prisoner = prisoner
```

Comment:
Really minor, this can be `req.journeyData.prisoner = await this.prisonerSearchService.getPrisonerDetails(req, csip.prisonNumber as string)` and that avoids intellij complaining about a redundant local variable
![image](https://github.com/user-attachments/assets/842dfe5a-467c-46c2-8647-4347ed350023)